sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/math.scala

```diff
@@ -202,3 +202,27 @@ case class Pow(left: Expression, right: Expression)
       """
   }
 }
+
+object Logarithm {
+  def apply(exprs: Seq[Expression]): Expression = {
```

Comment:
we don't need this one do we? just need an apply with one argument, along with the original case class' constructor.